src/main/java/com/qaprosoft/carina/demo/gui/pages/AutomationPracticeSearchResultPage.java

```diff
@@ -0,0 +1,33 @@
+package com.qaprosoft.carina.demo.gui.pages;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import com.qaprosoft.carina.demo.gui.components.AddedProductModal;
+import com.qaprosoft.carina.demo.gui.components.ProductCard;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class AutomationPracticeSearchResultPage extends AbstractPage {
+
+    @FindBy(xpath = "//div[@class='product-container']")
+    private ProductCard productCard;
```

Comment:
We can have more that one product when searching. Suggest to change it to the List<ProductCard>

Reply:
Ok, I will apply this fix!